google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -138,15 +137,12 @@ def create_finding(self, finding, source_id=None, finding_id=None):
                 '{}/findings/{}'.format(source_id, finding_id),
                 finding
             )
-            LOGGER.debug('Created finding response: %s',
+            LOGGER.debug('Successfully created finding response: %s',
                          response)
             return response
-        except (errors.HttpError, HttpLib2Error) as e:
-            LOGGER.exception(
-                'Unable to create CSCC finding: Resource: %s', finding)
-            violation_data = (
-                finding.get('source_properties').get('violation_data'))
-            raise api_errors.ApiExecutionError(violation_data, e)
+        except (errors.HttpError, HttpLib2Error):
+            LOGGER.debug('CSCC API exception encountered while creating '
```

Comment:
Per issue #2737, we only want to silence the specific errors for `409 "Requested entity already exists"`.  Other errors would still be handled as exceptions, and should not be silenced.

Reply:
Updated code accordingly!